shell/src/main/java/alluxio/shell/command/CpCommand.java

```diff
@@ -71,17 +79,42 @@ public void run(CommandLine cl) throws AlluxioException, IOException {
     String[] args = cl.getArgs();
     AlluxioURI srcPath = new AlluxioURI(args[0]);
     AlluxioURI dstPath = new AlluxioURI(args[1]);
-    List<AlluxioURI> srcPaths = AlluxioShellUtils.getAlluxioURIs(mFileSystem, srcPath);
-    if (srcPaths.size() == 0) {
-      throw new FileDoesNotExistException(
-          ExceptionMessage.PATH_DOES_NOT_EXIST.getMessage(srcPath.getPath()));
-    }
+    if (dstPath.getScheme() == null && "file".equals(srcPath.getScheme())) {
```

Comment:
I have addressed @aaudiber 's advise, use cp command to delegate copyToLocal and copyFromLocal command. 

So, I think CopyToLocalCommandTest, CopyFromLocalCommandTest and CpCommandTest can test the new cp command as comprehensive as it was.